processor/src/test/java/org/mapstruct/ap/testutil/runner/GeneratedSource.java

```diff
@@ -46,32 +46,15 @@
  */
 public class GeneratedSource implements TestRule {
 
+    private final static String FIXTURES_ROOT = "fixtures/";
```

Comment:
A checkstyle error crept in here. Otherwise it's okay to go. 👍
Could you correct this while squashing / merging?